starboard/shared/starboard/get_home_directory.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_SHARED_STARBOARD_GET_HOME_DIRECTORY_H_
+#define STARBOARD_SHARED_STARBOARD_GET_HOME_DIRECTORY_H_
+
+#include "starboard/shared/internal_only.h"
+
+namespace starboard {
+namespace shared {
+namespace starboard {
+
+// A platform implementation of getting the home directory.
```

Comment:
This wasn't clarified anywhere before, and a small nit, but perhaps add a short comment line to explain the implications of returning "false" from here.

Reply:
Also looks like most implementations default to `kSbSystemPathCacheDirectory` which is maybe also worth calling out.